modules/rosa-getting-started-create-cluster-admin-user.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-create-cluster-admin-user_{context}"]
+= Creating a cluster admin user for quick cluster access
+
+Before configuring an identity provider, you can create a user with `cluster-admin` privileges for immediate access to your {product-title} (ROSA) cluster.
+
+[NOTE]
+====
+The cluster admin user is useful when you need quick access to a newly deployed cluster. However, Red Hat recommends that you configure an identity provider and grant cluster administrator privileges to the identity provider users as required. For more information about setting up an identity provider for your ROSA cluster, see _Configuring an identity provider and granting cluster access_.
+====
+
+.Prerequisites
+
+* You have an AWS account.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a ROSA cluster.
+
+.Procedure
+
+. Create a cluster admin user:
++
+[source,terminal]
+----
+$ rosa create admin --cluster=<cluster_name> <1>
+----
+<1> Replace `<cluster_name>` with the name of your cluster.
++
+.Example output
+[source,terminal]
+----
+W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
+I: Admin account has been added to cluster '<cluster_name>'.
+I: Please securely store this generated password. If you lose this password you can delete and recreate the cluster admin user.
+I: To login, run the following command:
+
+   oc login https://api.example-cluster.wxyz.p1.openshiftapps.com:6443 --username cluster-admin --password d7Rca-Ba4jy-YeXhs-WU42J
+
+I: It may take up to a minute for the account to become active.
+----
++
+[NOTE]
+====
+It might take approximately one minute for the `cluster-admin` user to become active.
+====
+
+. Log in to the cluster through the CLI:
+.. Run the command provided in the output of the preceding step to log in:
++
+[source,terminal]
+----
+$ oc login <api_url> --username cluster-admin --password <cluster_admin_password> <1>
+----
+<1> Replace `<api_url>` and `<cluster_admin_password>` with the API URL and cluster admin password for your environment.
+.. Verify if you are logged in to the ROSA cluster as the `cluster-admin` user:
```

Comment:
I toggled between "that" and "if" when I wrote this and thought about both options. My view is that at this stage, the reader has not yet confirmed the result of the step. If they "verify that" they are logged in, then that to me seems to already imply that the result is successful. I used "verify if", because the result is as yet unknown.

Any feedback or further guidance about this would be really appreciated. Thanks!

Reply:
Based on our subsequent conversation in Slack, I will leave the line as "verify if".